### ISSUE TITLE
calculation of loss

### ISSUE THREAD

**Issue by merv22** [ku2482/fqf-iqn-qrdqn.pytorch#14]
Hi,
This is actually more or of question rather than a issue.

In the utils.py, line#39 where "quantile huber loss" is calculated, there is .detach() statement on the td_errors.
Could you please explain what is the reason?
Thanks

**Comment by toshikwa** [comment]
Sorry for the late response.

Gradients of operations like `x < 0` cloud be unstable, so I stopped the calculations of it.
Actually, this part is the "coefficient" of loss, so we don't need to calculate the gradient.

**Comment by merv22** [comment]
Thanks.
